src/nxt_conf_validation.c

```diff
@@ -2808,6 +2841,35 @@ nxt_conf_vldt_target(nxt_conf_validation_t *vldt, nxt_str_t *name,
 }
 
 
+#if (NXT_HAVE_CGROUP)
+
+static nxt_int_t
+nxt_conf_vldt_cgroup_path(nxt_conf_validation_t *vldt, nxt_conf_value_t *value,
+    void *data)
+{
+    char       path[PATH_MAX];
+    nxt_str_t  cgpath;
+
+    nxt_conf_get_string(value, &cgpath);
+    if (cgpath.length >= PATH_MAX - strlen(NXT_CGROUP_ROOT) - 1) {
+        return nxt_conf_vldt_error(vldt, "The cgroup name \"%V\" is too long.",
+                                   &cgpath);
+    }
+
+    sprintf(path, "/%*s/", (int) cgpath.length, cgpath.start);
+
+    if (strstr(path, "/./") != NULL || strstr(path, "/../") != NULL) {
```

Comment:
Do we need to reject `/./`?  I don't think it can hurt, since it's a no-op.

Reply:
It was more just mimicking what realpath(3) does and being consistent with disallowing '/../'...